### PR TITLE
INTYGFV-14621: Added tabs in header for user to navigate between pages

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

File: packages/common/src/components/AppHeader/AppHeaderTabs.test.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import AppHeaderTabs from './AppHeaderTabs'
+import { MemoryRouter, Route } from 'react-router-dom'
+import { Provider } from 'react-redux'
+import store from '@frontend/webcert/src/store/store'
+
+const getTabs = () => {
```
Comment (perfli) [inline packages/common/src/components/AppHeader/AppHeaderTabs.test.tsx:9]: We should add typings to the test

File: packages/common/src/components/AppHeader/AppHeaderTabs.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { UserTab } from '../../types/utils'
+import { CustomButton } from '@frontend/common'
+import styled from 'styled-components'
+import { useHistory, useRouteMatch } from 'react-router-dom'
+
+const Wrapper = styled.nav`
+  border: 1.5px solid rgba(1, 165, 163, 0.08);
```
Comment (perfli) [inline packages/common/src/components/AppHeader/AppHeaderTabs.tsx:8]: Don't we have predefined constants for the colors we use?

File: packages/webcert/src/App.tsx [hunk 3]
```diff
@@ -31,6 +31,7 @@ function App(): JSX.Element {
     window.addEventListener('beforeunload', handleWindowBeforeUnload)
     dispatch(cancelLogout())
     dispatch(getUser())
+    dispatch(getUserTabs())
```
Comment (perfli) [inline packages/webcert/src/App.tsx:34]: Do we need to wait on the data before letting the user interact with the app?

**Review by perfli — CHANGES_REQUESTED** [review]

File: packages/common/src/components/AppHeader/AppHeaderTabs.tsx (hunk 2 above)
Comment (mhornfeldt) [reply packages/common/src/components/AppHeader/AppHeaderTabs.tsx:8]: We tried using the css function var() before to access the colors. It worked locally but in the test env it didn't find the colors. Is there another way to do it?

Comment (perfli) [reply packages/common/src/components/AppHeader/AppHeaderTabs.tsx:8]: You could just define them in javascript, that's better than reproducing the values all over the code base.

File: packages/webcert/src/App.tsx (hunk 3 above)
Comment (mhornfeldt) [reply packages/webcert/src/App.tsx:34]: No

File: packages/common/src/components/AppHeader/AppHeaderTabs.tsx (hunk 2 above)
Comment (mhornfeldt) [reply packages/common/src/components/AppHeader/AppHeaderTabs.tsx]: Fixed

Comment (mhornfeldt) [reply packages/common/src/components/AppHeader/AppHeaderTabs.tsx]: Fixed

**Review by perfli — APPROVED** [review]

**Review by HelenaEdblad — APPROVED** [review]

File: packages/webcert/src/utils/userTabsUtils.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import { ResourceLink, ResourceLinkType, UserStatistics, UserTab } from '@frontend/common'
```
Comment (perfli) [inline packages/webcert/src/utils/userTabsUtils.tsx:1]: It's a bit redundant to have "Utils" in the file name `userTabsUtils`

**Pull request merged**